examples/ecs/system_closure.rs

```diff
@@ -1,6 +1,8 @@
 //! Shows how anonymous functions / closures can be used as systems.
 
-use bevy::{log::LogPlugin, prelude::*};
+use bevy::{ecs::schedule::IntoSystemDescriptor, log::LogPlugin, prelude::*};
+use bevy::ecs::schedule::SystemDescriptor;
+
```

Comment:
Is this needed?

Reply:
Oops, I accidentally pushed that from some experimentation of mine on something unrelated.